libs/mark-flow-ui/src/components/candidate_contest.tsx

```diff
@@ -92,30 +92,43 @@ export function CandidateContest({
   const [writeInCandidateModalIsOpen, setWriteInCandidateModalIsOpen] =
     useState(false);
   const [writeInCandidateName, setWriteInCandidateName] = useState('');
-  const [deselectedCandidate, setDeselectedCandidate] = useState('');
+  const [recentlyDeselectedCandidate, setRecentlyDeselectedCandidate] =
+    useState('');
+  const [recentlySelectedCandidate, setRecentlySelectedCandidate] =
+    useState('');
 
   const screenInfo = useScreenInfo();
 
   useEffect(() => {
-    if (deselectedCandidate !== '') {
+    if (recentlyDeselectedCandidate !== '') {
       const timer = setTimeout(() => {
-        setDeselectedCandidate('');
+        setRecentlyDeselectedCandidate('');
       }, 100);
       return () => clearTimeout(timer);
     }
-  }, [deselectedCandidate]);
+  }, [recentlyDeselectedCandidate]);
+
+  useEffect(() => {
+    if (recentlySelectedCandidate !== '') {
+      const timer = setTimeout(() => {
+        setRecentlySelectedCandidate('');
+      }, 100);
```

Comment:
If the `<AudioOnly>` element is removed while the audio is still being read out, the audio will still be allowed to finish, is that right? I imagine so given the video in the PR description - just double checking

Reply:
Yeah, good question - the screen reader does check to see if the element's still attached to the DOM before playing audio, but only at the beginning, so this just allows enough time for the audio to start, at which point the element can be removed without affecting playback

Reply:
Got it, thanks for explaining!